dcc-release-resources/src/main/resources/org/icgc/dcc/release/resources/mappings/gene-centric.mapping.json

```diff
@@ -49,6 +49,12 @@
               },
               "observation": {
                 "type": "nested"
+              },
+              "clinical_significance": {
+                "type": "object"
+              },
+              "clinical_evidence": {
+                "type": "nested"
```

Comment:
why does it need to be nested? Every nested object gets indexed as its own document and comes with its own overhead. Can this not just be an array?

Reply:
```
"clinical_evidence": {
            "civic": [
              {
                "evidenceLevel": "B"
              },
              {
                "evidenceLevel": "B"
              }
            ]
          }
```

In the future we may have more than just civic inside clinical_evidence so if it just smashed all the values together we would loose the relationship between the various evidence level and them belonging to the civic data set